### PR TITLE
Add test for show current offset's function

### PR DESCRIPTION
**Pull request by davendu**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Add a test for `afl.` command. See #455 and #456 for why the test is added.

**Test plan**

This test is intended to test if #456 works. A way to test is: checkout to 37712b4cb, the test should not pass. Checkout to latest `dev` and the test should work. Test command is: `cd test/ && rz-test db/cmd/cmd_af`.

**Closing issues**

N/A

### CODE REVIEW

**Comment by davendu** [comment]
Hi @ret2libc, sorry for latency, too busy. I just finished the test for `afl.`.

But I noticed some other tests are failing in `cmd/cmd_af`. Should it be better to also fix them in this PR, or wait until the analyzer is more stabilized? I think most of those errors are caused by the new analyzer generating new results.

Here is the test record (ignore the second failure, since it's the test for this PR):
```
root@rizin_docker:/rizin/test# rz-test db/cmd/cmd_af
Running from /rizin/test
Loaded 16 tests.
Skipping json tests because jq is not available.
[XX] /rizin/test/db/cmd/cmd_af af switch test
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'af
axf@0x100001308
' bins/mach0/mac-ls2
-- stdout
@@ -1,41 +1,41 @@
-c 0x1000012dc case.0x100001308.50
-c 0x10000130a case.0x100001308.49
-c 0x100001321 case.0x100001308.114
-c 0x10000132a case.0x100001308.120
-c 0x100001347 case.0x100001308.118
-c 0x100001353 case.0x100001308.99
-c 0x100001370 case.0x100001308.102
-c 0x100001392 case.0x100001308.97
-c 0x100001396 case.0x100001308.65
-c 0x1000013a2 case.0x100001308.84
-c 0x1000013b1 case.0x100001308.107
-c 0x1000013bd case.0x100001308.115
-c 0x1000013cc case.0x100001308.76
-c 0x1000013fb case.0x100001308.104
-c 0x10000140a case.0x100001308.80
-c 0x10000141a case.0x100001308.110
-c 0x10000143f case.0x100001308.108
-c 0x10000144b case.0x100001308.103
-c 0x100001473 case.0x100001308.83
-c 0x10000147f case.0x100001308.100
-c 0x100001492 case.0x100001308.111
-c 0x1000014d0 case.0x100001308.79
-c 0x1000014df case.0x100001308.113
-c 0x1000014f0 case.0x100001308.64
-c 0x1000014ff case.0x100001308.67
-c 0x100001519 case.0x100001308.82
-c 0x100001525 case.0x100001308.70
-c 0x10000153e case.0x100001308.66
-c 0x10000155b case.0x100001308.105
-c 0x10000156a case.0x100001308.119
-c 0x100001583 case.0x100001308.85
-c 0x1000015a0 case.0x100001308.87
-c 0x1000015ac case.0x100001308.71
-c 0x1000015c9 case.0x100001308.72
-c 0x1000015f7 case.0x100001308.112
-c 0x10000160d case.0x100001308.98
-c 0x10000162a case.0x100001308.116
-c 0x100001636 case.0x100001308.101
-c 0x100001645 case.0x100001308.109
-c 0x100001665 case.0x100001308.117
+c 0x1000012dc case.0x100001308.1
+c 0x10000130a case.0x100001308.0
+c 0x100001321 case.0x100001308.65
+c 0x10000132a case.0x100001308.71
+c 0x100001347 case.0x100001308.69
+c 0x100001353 case.0x100001308.50
+c 0x100001370 case.0x100001308.53
+c 0x100001392 case.0x100001308.48
+c 0x100001396 case.0x100001308.16
+c 0x1000013a2 case.0x100001308.35
+c 0x1000013b1 case.0x100001308.58
+c 0x1000013bd case.0x100001308.66
+c 0x1000013cc case.0x100001308.27
+c 0x1000013fb case.0x100001308.55
+c 0x10000140a case.0x100001308.31
+c 0x10000141a case.0x100001308.61
+c 0x10000143f case.0x100001308.59
+c 0x10000144b case.0x100001308.54
+c 0x100001473 case.0x100001308.34
+c 0x10000147f case.0x100001308.51
+c 0x100001492 case.0x100001308.62
+c 0x1000014d0 case.0x100001308.30
+c 0x1000014df case.0x100001308.64
+c 0x1000014f0 case.0x100001308.15
+c 0x1000014ff case.0x100001308.18
+c 0x100001519 case.0x100001308.33
+c 0x100001525 case.0x100001308.21
+c 0x10000153e case.0x100001308.17
+c 0x10000155b case.0x100001308.56
+c 0x10000156a case.0x100001308.70
+c 0x100001583 case.0x100001308.36
+c 0x1000015a0 case.0x100001308.38
+c 0x1000015ac case.0x100001308.22
+c 0x1000015c9 case.0x100001308.23
+c 0x1000015f7 case.0x100001308.63
+c 0x10000160d case.0x100001308.49
+c 0x10000162a case.0x100001308.67
+c 0x100001636 case.0x100001308.52
+c 0x100001645 case.0x100001308.60
+c 0x100001665 case.0x100001308.68
 c 0x100001682 case.default.0x100001308


[XX] /rizin/test/db/cmd/cmd_af af display current offset function
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 's main
af
afl.
s. 1
afl.
' bins/elf/ls
-- stderr
Warning: run rizin with -e io.cache=true to fix relocations in disassembly

-- exit status: -1
[XX] /rizin/test/db/cmd/cmd_af af localflag
RZ_NOPLUGINS=1 rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e analysis.calls=false
af
aflc
e analysis.calls=true
af-*
af
aflc
' bins/mach0/mac-ls2
-- stdout
@@ -1,2 +1,2 @@
 1
-33
+35


[16/16]                    10 OK         3 BR        3 XX        0 FX
Finished in 0 seconds.
```

**Comment by ret2libc** [comment]
> Hi @ret2libc, sorry for latency, too busy. I just finished the test for afl..

No problem ;) And thanks

> But I noticed some other tests are failing in cmd/cmd_af. Should it be better to also fix them in this PR, or wait until the analyzer is more stabilized? I think most of those errors are caused by the new analyzer generating new results.

There is no existing error in cmd_af. As you can see the CI is green, that means all tests are passing. How did you experience those errors?

**Comment by davendu** [comment]
> As you can see the CI is green, that means all tests are passing

I just noticed that....strange, my local test has tons of failure. Maybe my local test bin is different from the remote one and my local errors are false alarms.

Anyway, since the PR have passed, let me submit this draft in a minute ;)


--------------------------------------------

Daven Du(杜岳峰)
Keenlab of Tencent, CSIG

File: test/db/cmd/cmd_af [hunk 1]
```diff
@@ -15,6 +15,21 @@ EXPECT=<<EOF
 EOF
 RUN
 
+NAME=af display current offset function
+FILE=bins/elf/ls
+CMDS=<<EOF
+s main
+af
+afl.
+s. 1
```
Comment (ret2libc) [inline test/db/cmd/cmd_af]: I think you should use sd and not s., Just to be sure and clearer

Comment (ret2libc) [inline test/db/cmd/cmd_af]:
```suggestion
sd 1
```

Comment (davendu) [reply test/db/cmd/cmd_af]: Oh agree! did not noticed there was a comment on PR......
Thanks a lot for help!

Comment (ret2libc) [reply test/db/cmd/cmd_af]: no problem ;)

**Pull request merged**